src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiArchiveInfo.java

```diff
@@ -453,7 +454,7 @@ public boolean equals( Object o )
 
         KojiArchiveInfo that = (KojiArchiveInfo) o;
 
-        return getArchiveId() == that.getArchiveId();
+        return Objects.equals(getArchiveId(), that.getArchiveId());
```

Comment:
Should we change `hashCode` to `Objects.hash` (or is it `hashCode`)? The current implementation doesn't look null safe, in any case.

Reply:
Done